pkgs/development/libraries/qt-6/patches/qtbase-qmake-pkg-config.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/qmake/generators/makefile.cpp b/qmake/generators/makefile.cpp
+--- a/qmake/generators/makefile.cpp
++++ b/qmake/generators/makefile.cpp
+@@ -3390,8 +3390,7 @@ MakefileGenerator::writePkgConfigFile()
+       << varGlue("QMAKE_PKGCONFIG_CFLAGS", "", " ", " ")
+         //      << varGlue("DEFINES","-D"," -D"," ")
+          ;
+-    if (!project->values("QMAKE_DEFAULT_INCDIRS").contains(includeDir))
+-        t << "-I${includedir}";
```

Comment:
What are the `QMAKE_DEFAULT_INCDIRS`, shouldn't they be included by default rendering the added `-I` flag useless.

Reply:
Same issue as #52457